tests/caasp/first_boot.pm

```diff
@@ -27,6 +27,12 @@ sub run {
     }
 
     microos_login;
+    if (check_var('DISTRI', 'kubic')) {
+        assert_script_run("for repo in \$(zypper lr -U | grep download.opensuse.org | awk '{ print \$1 }'); do zypper mr -d \$repo; done");
+        my $mirror = get_var('MIRROR_HTTP');
```

Comment:
Maybe use `get_required_var` in case it's not defined

Reply:
Sure, this makes sense.